docs/src/use/configure/rules.md

```diff
@@ -277,31 +305,47 @@ console.log('hello');
 
 ### Using configuration files
 
-To disable rules inside of a configuration file for a group of files, use the `overrides` key along with a `files` key. For example:
+To disable rules inside of a configuration file for a group of files, use a subsequent config object with a `files` key. For example:
 
-```json
-{
-  "rules": {...},
-  "overrides": [
+```js
+// eslint.config.js
+export default [
+    {
+        rules: {
+            "no-unused-expressions": "error"
+        }
+    },
     {
-      "files": ["*-test.js","*.spec.js"],
-      "rules": {
+      files: ["*-test.js","*.spec.js"],
+      rules: {
         "no-unused-expressions": "off"
       }
     }
-  ]
-}
+];
 ```
 
 ### Disabling Inline Comments
 
 To disable all inline config comments, use the `noInlineConfig` setting in your configuration file. For example:
 
-```json
-{
-  "rules": {...},
-  "noInlineConfig": true
-}
+```js
+// eslint.config.js
+export default [
+    {
+        linterOptions: {
+            noInlineConfig: true
+        },
+        rules: {
+            "no-unused-expressions": "error"
+        }
+    },
+    {
+      files: ["*-test.js","*.spec.js"],
+      rules: {
+        "no-unused-expressions": "off"
+      }
+    }
+];
```

Comment:
```suggestion
    }    
];
```

I think this is a copy & paste error, this doesn't seem relevant for this example.